frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -236,8 +241,30 @@ export const insightLogic = kea<insightLogicType>({
                     for (const key of Object.keys(metadata)) {
                         updatedInsight[key] = response[key]
                     }
+
                     savedInsightsLogic.findMounted()?.actions.loadInsights()
                     dashboardsModel.actions.updateDashboardInsight(updatedInsight)
+
+                    lemonToast.success(`Updated insight`, {
+                        button: {
+                            label: 'Undo',
+                            action: async () => {
+                                const response = await api.update(
+                                    `api/projects/${teamLogic.values.currentTeamId}/insights/${values.insight.id}`,
```

Comment:
Can we do `api.get.the.url.somehow()`?

Reply:
From laziness and focus perspective. All of the URLs in this logic are boutique and locally-sourced. Let's tidy them in one go :)